crates/biome_deserialize/src/visitor.rs

```diff
@@ -3,34 +3,26 @@ use biome_rowan::{Language, SyntaxNode};
 
 /// Generic trait to implement when resolving the configuration from a generic language
 pub trait VisitNode<L: Language>: Sized {
-    /// Called when visiting the key of a member
-    fn visit_member_name(
+    /// Called when visiting a plain value, i.e. a value that is not a list.
+    fn visit_plain_value(
         &mut self,
         _node: &SyntaxNode<L>,
         _diagnostics: &mut Vec<DeserializationDiagnostic>,
     ) -> Option<()> {
-        unimplemented!("you should implement visit_member_name")
-    }
-    /// Called when visiting the value of a member
-    fn visit_member_value(
-        &mut self,
-        _node: &SyntaxNode<L>,
-        _diagnostics: &mut Vec<DeserializationDiagnostic>,
-    ) -> Option<()> {
-        unimplemented!("you should implement visit_member_value")
+        unimplemented!("you should implement visit_plain_value")
     }
```

Comment:
This crate is meant to create "events" on specific types so the consumer can do what they need on those types. By removing these visitors, we removed the ability to visit them. I don't think that's what we meant. It's like a simplified `serde`.

While `member_name` and `member_value` can arguably be better names, `plain_value` is more cryptic because it doesn't explain what we are visiting. Value of a key? And what about the keys themselves?